ui/src/components/header/header.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { Button, PageHeader, Row, Typography } from 'antd';
+import { useAccount, useMsal } from "@azure/msal-react";
+import './header.less';
+
+type Props = {};
+
+const Header: React.FC<Props> = () => {
+  const { accounts, instance } = useMsal();
+  const account = useAccount(accounts[0] || {});
+  const [name, setName] = useState("");
+
+  useEffect(() => {
+    if (account && account.name) {
+      setName(account.name.split(" ")[0]);
+    }
+  }, [account]);
+
+  const onClickLogout = () => {
+    instance.logoutRedirect().catch(e => {
+      console.error(e);
+    });
+  }
+
+  const { Paragraph } = Typography;
+  return (
+    <PageHeader
+      title={ `Hello ${ name }, welcome to Azure Feature Store` }
```

Comment:
Feathr?

Reply:
Changed Azure Feature Store to Feathr Feature Store